posts/coding-intentionally-in-bash-grains.md

```diff
@@ -0,0 +1,194 @@
+**Spoiler Alert: This article has spoilers for the Grains exercise in general and specifically the Grains exercise on the Bash track.  If you haven't completed it yourself, yet, and you don't want to be shown some solutions, come back once you've finished it!**
+
+It's your first day at a new company.  You've done all the paperwork, met the team, and it is finally time for you to sit down and start reading some of the code that you'll be working on.  You start to read through the different functions, classes, and modules, and, as you read, you find yourself starting to squint at the screen in confusion.  You keep reading and single word escapes from your mouth, barely spoken, almost breathed: "Whaaaaaaaaaat..."[^1]  The more you go on, the more this happens as you get more bewildered and even a little angry.  
+
+> What is happening in this code?  It's so complex, and it's *way* different from how I usually write my code!
```

Comment:
We have *way* twice in two paragraphs now :)

How about just stopping it short after the first question? I feel like you've really built up the emotions and situation in the paragraph, so the direct quote feels like it is less important.

```suggestion
> What is happening in this code?
```